.changeset/shiny-geckos-hang.md

```diff
@@ -0,0 +1,6 @@
+---
+"@sumup-oss/circuit-ui": minor
+---
+
+Marked the `Calendar` component as stable.
```

Comment:
Component names aren't quoted by convention (I know this is inconsistent from the icon names 🙈)

```suggestion
Marked the Calendar component as stable. Update the related imports:

```diff
-import { Calendar, CalendarProps, PlainDateRange } from '@sumup-oss/circuit-ui/experimental;
+import { Calendar, CalendarProps, PlainDateRange } from '@sumup-oss/circuit-ui;
```
```

Reply:
Make sure to include the closing backticks for the diff block, GitHub is confused by the nested code block.